P5/Exemplars/tei_jtei.odd

```diff
@@ -2969,7 +2976,9 @@
                 <sch:let name="apos.straight" value="''''"/>
                 <sch:let name="quotes" value="concat('[', $apos.straight, '&quot;]')"/>
                 <sch:let name="div.types.front" value="('abstract', 'acknowledgements', 'authorNotes', 'editorNotes', 'corrections', 'dedication')"/>
-                <sch:let name="tei.version.url" value="'http://www.tei-c.org/release/doc/tei-p5-doc/VERSION'"/>
+                <!-- http://www.tei-c.org/release/doc/tei-p5-doc/VERSION produces 
+                     429 "Too Many Requests" errors -->
+                <sch:let name="tei.version.url" value="'https://jenkins.tei-c.org/view/TEI%20master/job/TEIP5/lastStableBuild/artifact/P5/release/doc/tei-p5-doc/VERSION/*view*/'"/>
```

Comment:
Better strip off the 'view/TEI%20master'-part: `https://jenkins.tei-c.org/job/TEIP5/lastStableBuild/artifact/P5/release/doc/tei-p5-doc/VERSION/*view*/`

Reply:
Thanks, I see it works even without the `/*view*/` part, so I'll change to https://jenkins.tei-c.org/job/TEIP5/lastStableBuild/artifact/P5/release/doc/tei-p5-doc/VERSION.